### ISSUE TITLE
Cannot import rrule library: TypeError: Cannot read property 'HOURLY' of undefined

### ISSUE THREAD

**Issue by blogui91** [jakubroztocil/rrule#430]
- [X] Verify that you've looked through existing issues for duplicates before
  creating a new one
- [X] Code sample reproducing the issue. Be sure to include all input values you
  are using such as the exact RRule string and dates.
  ```
      import { RRule } from 'rrule';

      console.log(RRule);
 
- [X] Expected output
   I should see a log with the RRule value
- [X] Actual output
![image](https://user-images.githubusercontent.com/5723380/96193163-3601a100-0f0d-11eb-9be8-f978ef2d5219.png)

- [X] The version of `rrule` you are using
 "rrule": "^2.6.6"
- [X] Your operating system
MacOS 
- [X] Your local timezone
Thu Oct 15 19:42:45 CDT 2020 (America/Monterrey)
 
Basically, I am trying to log the RRule class but i got the error above.

**Comment by olemartinorg** [comment]
I've done some research on this, as I'm getting the same error. It's baffling to me that this would crash on load, so i suspect we're in the minority with this problem.

I get this problem using `rollup-plugin-commonjs-alternate`, but if I switch to `@rollup/plugin-commonjs` everything seems to work as expected. I see @blogui91 is using webpack, which might have the same problem, but I haven't seen any details on the build setup, and I'm less familiar with webpack, so I was unable to reproduce using it.

From the emitted code, it seems this is caused by the [lazy, runtime import to avoid circular refs](https://github.com/jakubroztocil/rrule/blob/d37d88a7eedf2cbcb0e39729ec12871d5c006f11/src/rrule.ts#L22). For me, `rollup-plugin-commonjs-alternate` correctly identifies that `nlp/index.js` should be imported when encountering the runtime import, but `@rollup/plugin-commonjs` fails to handle the import at all, so rrule is missing the needed code (and so it does not produce the error).

Properly resolving the circular imports should solve this. I'll take a look at it.

**Comment by sciyoshi** [comment]
This is likely fixed in the new version of rrule.
